docs/rules/no-extraneous-dependencies.md

```diff
@@ -1,4 +1,6 @@
-# import/no-extraneous-dependencies: Forbid the use of extraneous packages
```

Comment:
preserve

Reply:
There's already a sentence "Forbid the import of external modules that are not declared..." that should capture this.

Reply:
Per https://github.com/import-js/eslint-plugin-import/pull/2582#discussion_r1008920390, i think the exact description phrase text needs to appear in the doc somewhere.